content/backmatter.tex

```diff
@@ -490,6 +493,15 @@ \subsection{SMA\_* Environment Variables}
 were deprecated in order to normalize the \openshmem \ac{API} to use
 \shmemprefix{} as the standard prefix for all environment variables.
 
+\subsection{Fortran API}\label{subsec:deprecate-fortran}
+The entire \openshmem \Fortran API was deprecated due to a general lack of
```

Comment:
Replace "due to" with "because of".

Ref: https://english.stackexchange.com/questions/7385/can-due-to-and-because-of-be-used-interchangeably